core/src/organization/member-restriction/member-restriction.resolver.ts

```diff
@@ -46,4 +51,10 @@ export class MemberRestrictionResolver {
     async member(@Root() memberRestriction: Partial<MemberRestriction>): Promise<Member> {
         return memberRestriction.member ?? await this.memberService.getMemberById(memberRestriction.memberId!);
     }
+
+    @Subscription(() => MemberRestrictionEvent)
+    async followBannedMembers(): Promise<AsyncIterator<MemberRestrictionEvent>> {
```

Comment:
`followRestrictedMembers` unless referring to queue bans, otherwise add that to the name?

Reply:
Sure.